.mdox.validate.yaml

```diff
@@ -42,6 +42,5 @@ validators:
     type: 'ignore'
   - regex: 'twitter\.com'
     type: 'ignore'
-  # 500 when requested my mdox in GH actions.
-  - regex: 'outshift\.cisco\.com'
+  - regex: 'outshift\.cisco\.com\/blog\/multi-cluster-monitoring'
```

Comment:
The domain works fine, only `/blog/multi-cluster-monitoring` has a [problem](https://github.com/thanos-io/thanos/pull/7429#issuecomment-2168018338).